src/main/kotlin/io/github/enyason/predictions/RetrofitFactory.kt

```diff
@@ -0,0 +1,38 @@
+package io.github.enyason.predictions
+
+import com.google.gson.GsonBuilder
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object RetrofitFactory {
```

Comment:
can we take this factory out of the `predictions` package since it does not only apply to predictions?

RetrofitFactory is not a part of predictions.